libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -1024,6 +1024,17 @@
     <output name="out" type="float" />
   </nodedef>
 
+  <!--
+    Node: <checkers2d>
```

Comment:
Would `checkerboard` perhaps be the more canonical name for this node?  I can see the advantage in leaving MaterialX open to similar 3D patterns, but after all a checkerboard is definitively 2D.  :)

Reply:
That sounds like a good idea to me, thanks @jstone-lucasfilm